src/test/java/tests/FeignREST.java

```diff
@@ -0,0 +1,57 @@
+package tests;
+
+import data.Parameters;
+import enums.ContentTypes;
+import enums.FileNames;
+import enums.Paths;
+import enums.Statuses;
+import feign.Response;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import utils.ApiUtils;
+import utils.JsonPlaceholder;
+import utils.PropertyUtils;
+import utils.StringUtils;
+
+import java.io.IOException;
+
+public class FeignREST extends BaseTest {
```

Comment:
for test class we need to specify Test word in the end of name
e g FeignRESTTest